ng-schematics fails by default

```
~/src/my-first-project main
❯ ng e2e
Cannot find "e2e" target for the specified project.
You can add a package that implements these capabilities.

For example:
  Cypress: ng add @cypress/schematic
  Nightwatch: ng add @nightwatch/schematics
  WebdriverIO: ng add @wdio/schematics
  Puppeteer: ng add @puppeteer/ng-schematics

Would you like to add a package with "e2e" capabilities now? Puppeteer
ℹ Using package manager: npm
✔ Found compatible package version: @puppeteer/ng-schematics@0.5.0.
✔ Package information loaded.

The package @puppeteer/ng-schematics@0.5.0 will be installed and executed.
Would you like to proceed? Yes
✔ Packages successfully installed.
? Which test runners do you wish to use? Use Node Test Runner [https://nodejs.org/api/test.html]
CREATE e2e/.gitignore (35 bytes)
CREATE e2e/tsconfig.json (193 bytes)
CREATE e2e/tests/app.test.ts (398 bytes)
CREATE e2e/tests/utils.ts (881 bytes)
UPDATE package.json (1165 bytes)
UPDATE angular.json (3007 bytes)
✔ Packages installed successfully.
✔ Packages installed successfully.

~/src/my-first-project main* 30s
❯ ng e2e

 Building tests 🛠️ ...

node_modules/puppeteer/lib/types.d.ts:4:10 - error TS2305: Module '"mitt"' has no exported member 'EventType'.

4 import { EventType } from 'mitt';
           ~~~~~~~~~


Found 1 error in node_modules/puppeteer/lib/types.d.ts:4


 🛑 Test failed! 🛑
Please look at the output above to determine the issue!

~/src/my-first-project main*
❯ ng e2e

 Building tests 🛠️ ...

node_modules/puppeteer/lib/types.d.ts:4:10 - error TS2305: Module '"mitt"' has no exported member 'EventType'.

4 import { EventType } from 'mitt';
           ~~~~~~~~~


Found 1 error in node_modules/puppeteer/lib/types.d.ts:4


 🛑 Test failed! 🛑
Please look at the output above to determine the issue
```

--- a/packages/ng-schematics/src/schematics/ng-add/files/common/e2e/tests/app.__ext@dasherize__.ts.template
+++ b/packages/ng-schematics/src/schematics/ng-add/files/common/e2e/tests/app.__ext@dasherize__.ts.template
@@ -10,7 +10,7 @@ describe('App test', function () {
   setupBrowserHooks();
   it('is running', async function () {
     const {page} = getBrowserState();
-    const element = await page.waitForSelector('text/<%= project %> app is running!');
+    const element = await page.waitForSelector('text/<%= project %>');
 
 <% if(testRunner == 'jasmine' || testRunner == 'jest') { %>
     expect(element).not.toBeNull();

